ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -81,21 +81,27 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-single-node-workers
-  cron: 12 9 8,24 * *
+- as: e2e-aws-ovn-single-node-workers-serial
+  interval: 168h
   steps:
     cluster_profile: aws-2
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
+      TEST_SUITE: openshift/conformance/serial
+    observers:
+      enable:
+      - observers-resource-watch
     workflow: openshift-e2e-aws-single-node-workers
-  timeout: 3h0m0s
-- as: openshift-e2e-aws-single-node-workers-upgrade-conformance
-  cron: 58 4 8,23 * *
+- as: e2e-aws-ovn-single-node-workers
+  interval: 168h
   steps:
     cluster_profile: aws-2
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
-    workflow: openshift-e2e-aws-single-node-workers-upgrade-conformance
```

Comment:
is the upgrade conformance now implicit?

Reply:
Yeah same way the openshift-e2e-aws-single-node has conformance as implicit